conda_smithy/templates/README.md.tmpl

```diff
@@ -71,10 +71,22 @@ Current build status
 ====================
 {%- set appveyor_badge_name = github.repo_name.replace('_', '-') %}
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
-
+{# #}
+{%- if "circle" in ci_enabled %}
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
+{%- else %}
+Linux: ![](https://cdn.rawgit.com/{{github.user_or_org}}/{{github.repo_name}}/master/ci_support/disabled.svg)
```

Comment:
So the one thing that I don't love about that solution is there is a lag between when the file gets moved and when a feedstock is re-rendered. We may decide to move the file, but not create a release. The net result being that all feedstocks have a temporary outage for this badge.

If we pick a commit, it won't have this problem. If we change the location and commit in conda-smithy, it will also be updated when we re-render. Though it will have no effect in-between merging and re-rendering.